### ISSUE TITLE
Remix plugin: Expand new receipt after submitting a contract

### ISSUE THREAD

**Issue by manuelwedler** [ethereum/sourcify#1754]
Another thing I realised when watching [this video](https://youtu.be/lRo-TBLTgzs?t=1914): When the user is moved over to the receipt view after submitting a contract, the status of the verification is only displayed after expanding the receipt. I think this should happen automatically, otherwise the user might be unaware of a failed verification.

So when a contract was submitted, we should automatically move over to the receipt view and expand the new receipt.

**Comment by kuzdogan** [comment]
Agreed, great catch. Just a reminder if you want to move this to Candidates or not.
